check_env.ipynb

```diff
@@ -75,21 +75,21 @@
  "metadata": {
   "anaconda-cloud": {},
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```

Comment:
It only worked with Python 3 last year, but now it works with Python 2 too?

Reply:
Good catch -- it should work with Python 2 on Linux and macOS, but we should keep this as `python3`. Will fix...